docs/dev_guide/how_to/workspace_setup.md

```diff
@@ -7,16 +7,31 @@ permalink: /docs/dev_guide/how_to/workspace_setup
 ---
 
 # Venice Workspace Setup
+We recommend using a Unix-based environment for development, such as Linux or macOS.
+If you're on Windows, we recommend using [WSL](https://learn.microsoft.com/en-us/windows/wsl/install).
```

Comment:
wsl or wsl2?

Reply:
I can update it to wsl2. I've heard the experience is much better.